src/chapters/intro-to-data/intro-to-data.rst

```diff
@@ -1,24 +1,30 @@
 Introduction To Data
 ====================
 
-Readings
---------
-
 Read the following articles, follow along where instructed:
 
-| `Data Analysis Methods & Techniques <https://www.datapine.com/blog/data-analysis-methods-and-techniques/>`__.
+Methods and Techniques
+----------------------
+
+1. `Data Analysis Methods & Techniques <https://www.datapine.com/blog/data-analysis-methods-and-techniques/>`__.
 
 * Key Takeaways: Video about qualitative vs quantitative data illustrates some key points!
-  
-| `A Beginner's Guide to Data Analysis in Python <https://towardsdatascience.com/a-beginners-guide-to-data-analysis-in-python-188706df5447>`__.
+
+Data Analytics in Python
+------------------------
+
+2. `A Beginner's Guide to Data Analysis in Python <https://towardsdatascience.com/a-beginners-guide-to-data-analysis-in-python-188706df5447>`__.
 
```

Comment:
Under each new section ordered lists should restart and not continue from the previous section.

Use #. in front of link insted of hard coding.